tests/openapi/test_service.py

```
@@ -58,8 +60,42 @@ def test_telemetry_detailed():
 
     endpoint = result['requests']['rest']['responses']['PUT /collections/{name}/points']
     assert endpoint['200']['count'] > 0
+    
+    if level == 0:
+        'collections' not in result['collections']
```

Ruff (0.8.2)

Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)